CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Changes
 
+## edge-23.6.3
+
+This edge release adds support to HA mode for the multicluster service-mirror
```

Comment:
We're still missing the HA flag :( We only have leader election working for now, but no way to set the replicas. Should we omit this?

Reply:
I've reworded the intro and the bullet point to provide more details. Let me know if that makes sense.

Reply:
yup! looks great